CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 ------
 
 **ENHANCEMENTS**
+- Add the configuration parameter `DefaultUserHome` under `HeadNode` to allow users to move the default user's home directory to `/local/home` instead of `/home` (default).
```

Comment:
*Minor*
If you want to specify the section where the parameter is placed (good practice), we should write "configuration parameter `DeploymentSettings/DefaultUserHome`"